src/store/consts/utils.ts

```diff
@@ -253,6 +254,27 @@ export const calculateTimeRemaining = (slot: number, nextRoundStartSlot: BN): BN
   }
   return slotDiff.muln(slotTime)
 }
+export const getAddressFromIndex = (
```

Comment:
rename this function to 'getSyntheticAndCollateralAdresses', current name isn't exactly understandable